lib/AzModuleInstaller.js

```diff
@@ -0,0 +1,168 @@
+"use strict";
```

Comment:
i don't think we need to check in the js compiled files into master. pls check with @aksm-ms

Reply:
I agree it doesn't matter but since these are already checked in so I added as well.

Reply:
Checked with @aksm-ms and she said to checkin js as well to master similar to the existing code.